tests/Data/ProgrammesDb/EntityRepository/CoreEntityRepository/FindProgrammesByAncestryAndTypesTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace Tests\BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository;
+
+use Tests\BBC\ProgrammesPagesService\AbstractDatabaseTest;
+
+/**
+ * @covers BBC\ProgrammesPagesService\Data\ProgrammesDb\EntityRepository\CoreEntityRepository::<public>
+ * @group repository
+ * @group programmesAggregation
+ */
+class FindProgrammesByAncestryAndTypesTest extends AbstractDatabaseTest
+{
+    public function testProgrammesChildrenGetCorrectClips()
+    {
+        $this->loadFixtures(['MongrelsFixture']);
+
+        $dbAncestryId = [1, 15];
```

Comment:
yeah this could lead to potentially flaky tests. Take a look at this. Use that getAncestry function to get the ancestry instead of using the ids straight away